### PR TITLE
Adding some style to the project

### PR DESCRIPTION
**Pull request by Bigizi**
**Leader-Board-Project**

I added some styles to the project in the style CSS file.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -8,24 +8,26 @@
...
 <body>
     <main class="container">
-        <section>
-            <h1>Leaderboard</h1><br/>
-            <div class="section-side1">
-                <p id="parag">Recent scores</p>
-                <button class="btn-1">Refresh</button>
-            </div><br/>
-            <div class="section-side2">
+        <h1 id="h1">LeaderBoard</h1>
```
Comment (bobb-Rob) [inline dist/index.html:11]: - [ ] Your README file does look professional to some level but needs to contain all that's needed for running the app with the API(s) it consumes. It should have instructions about how to clone, install, and any other details needed to run the app. This information should be provided knowing that both non-technical and technical users will use it.
- [ ] Please add license section of this README. Ensure when clicked, it take the user to the license page on github. 

You README.md **_Getting started section_**
image

Please edit this section showing how one can use this repo.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @Bigizi , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -134,3 +133,11 @@ If you like this project...
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 > I would like to thank all my coding partners, morning session team and stand up call team for the support.
 <p align="right">(<a href="#acknowledgements">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (GzimAsani) [inline README.md]: - [ ] The MIT link is returning a 404 page, kindly update it 
image

File: README.md [hunk 3]
```diff
@@ -76,13 +76,11 @@ In order to run this project you need:
...
-### Install
-
-Install any code editor in your computer
+git clone [https://github.com/Bigizi/LeaderBoard]
 
-clone the repo using this command:
+### Install
 
-git clone []
+Install any code editor in your computer the follow the step above to clone the repository
```
Comment (GzimAsani) [inline README.md]: - [ ] As asked by the previous reviewer please include on your readme file how a non-technical user can clone, install dependencies and run your project, include commands as well

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @Bigizi                         :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with adding some of the requirements asked previously :+1: 

 ## Required Changes 
- [ ] Check the comments below :arrow_double_down: 


 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Crispin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**